mozilla/kuma
# Bug 1139490: Manage all data URIs with Pipeline

### Testing

1. Enter production mode
2. Verify that Tabzilla still looks good
3. Visit [this page](https://developer-local.allizom.org/en-US/users/signin?next=/en-US/docs/new) while logged out and verify that the Persona icon still looks good
